tests/pyreverse/test_writer.py

```diff
@@ -110,22 +114,22 @@ def _setup(project, config, writer):
 @pytest.mark.usefixtures("setup_dot")
 @pytest.mark.parametrize("generated_file", DOT_FILES)
 def test_dot_files(generated_file):
-    expected_file = os.path.join(os.path.dirname(__file__), "data", generated_file)
-    generated = _file_lines(generated_file)
-    expected = _file_lines(expected_file)
-    generated = "\n".join(generated)
-    expected = "\n".join(expected)
-    files = f"\n *** expected : {expected_file}, generated : {generated_file} \n"
-    diff = "\n".join(
-        line for line in unified_diff(expected.splitlines(), generated.splitlines())
-    )
-    assert expected == generated, f"{files}{diff}"
-    os.remove(generated_file)
+    _assert_files_are_equal(generated_file)
```

Comment:
The logic for each test case is basically the same, but the used fixtures differ. To avoid duplicated code I extracted a helper function for it. 
The ``setup_xy`` are fixtures as they 
a) shall perform cleanup code after the test case finishes
b) test scope is comparing the output file with expectation, not the process of generating the files itself

Is there a way to specify the fixture to use in ``pytest.mark.parametrize``?